README.md

```diff
@@ -1,3 +1,29 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
+
+- [Local Font Access Explained](#local-font-access-explained)
```

Comment:
Is there a way to make doctoc skip H1 items and only do H2...H4 ? Since there's only a single H1 in the doc (used as a title), this looks a bit funny.